packages/react-flex/package.json

```diff
@@ -35,7 +35,12 @@
     "react-dom": "16.8.6"
   },
   "dependencies": {
+    "@fluentui/react-bindings": "^0.51.0",
+    "@fluentui/react-button": "^0.7.12",
+    "@fluentui/react-compose": "^0.12.7",
+    "@fluentui/react-theme-provider": "^0.4.2",
     "@uifabric/set-version": "^7.0.19",
+    "office-ui-fabric-react": "^7.124.2",
```

Comment:
Is the reason we need this dependency because of the examples?

Reply:
Yes, we're going to need to find a way around that all-up very soon... :(

Reply:
yes, I could make some temporary components for the examples if it's better.